### PR TITLE
Add missing variable available to the documentation

### PR DESCRIPTION
**Pull request by unfunco**
Added missing variable to documentation.

### CODE REVIEW

**Comment by unfunco** [comment]
I did fix the other branch much easier to rebase by using the `--onto` flag, however, because that branch has now been `+`pushed...

screen shot 2016-12-23 at 09 05 42

So this branch will need to be the one that's merged (assuming you want it merged) and #1047 can remain closed.

File: docs/cookbook/templates.rst [hunk 1]
```diff
@@ -80,6 +80,7 @@ template you are overriding:
    - ``%name%``  the specification name
    - ``%namespace%`` the specification namespace
    - ``%subject%`` the name of the class being specified
+   - ``%subject_class%`` the name of the subject class being specified
```
Comment (jakzal) [inline docs/cookbook/templates.rst]: The description is not really clear. Subject's description could be also clarified. How about:

```
- ``%subject%``the fully qualified name of the class being specified
- ``%subject_class%`` the name of the class being specified
```

Comment (ciaranmcnulty) [inline docs/cookbook/templates.rst]: Yep I agree

**Comment by ciaranmcnulty** [comment]
Thanks!

**Pull request merged**